examples/article_info.py

```diff
@@ -25,3 +25,9 @@
 print(first_step.title)
 print(first_step.description)
 print(first_step.picture)
+
+print(article.co_authors)
```

Comment:
Shouldn't we keep all print statements of article data members together? After line 20.

Reply:
Okay, I will do that.